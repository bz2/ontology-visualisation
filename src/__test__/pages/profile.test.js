import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Index from '../../pages/index'

const setup = () => {
  const props = {
    updateStoreValue: jest.fn(),
    user: {
      email: 'a@b.c'
    }
  }
  const component = shallow(<Index {...props} />)

  return {
    component
  }
}

describe('Profile page', () => {
  afterEach(() => {
    jest.clearAllMocks()
  })

  it('should match snapshot', () => {
    const { component } = setup()
    expect(toJson(component)).toMatchSnapshot()
  })
})
