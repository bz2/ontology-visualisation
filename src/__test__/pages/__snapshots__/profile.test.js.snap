// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile page should match snapshot 1`] = `
<Connect
  actions={
    Object {
      "addSubValueToObject": [Function],
      "addToArray": [Function],
      "addToObject": [Function],
      "removeFromArray": [Function],
      "removeFromObject": [Function],
      "setStoreState": [Function],
    }
  }
  addToObject={[MockFunction]}
  mapToProps={[Function]}
  user={
    Object {
      "email": "a@b.c",
    }
  }
>
  <Component />
</Connect>
`;
